### PR TITLE
net-dns/maradns:  add version 2.0.17

### PR DESCRIPTION
**Pull request by ktrace**
Add new version 2.0.17, last on 2.x branch.

Package-Manager: Portage-2.3.76, Repoman-2.3.16
Signed-off-by: Victor Kustov <ktrace@yandex.ru>
Closes: https://github.com/gentoo/gentoo/pull/13381

### CODE REVIEW

File: net-dns/maradns/maradns-2.0.17.ebuild [hunk 1]
```diff
@@ -0,0 +1,111 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 )
```
Comment (juippis) [inline net-dns/maradns/maradns-2.0.17.ebuild]: Is there no chance for python3 support? We are currently looking into dropping python2.

File: net-dns/maradns/maradns-2.0.17.ebuild [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+inherit python-single-r1 systemd toolchain-funcs user
+
+DEADWOOD_VER="3.2.14"
+
+DESCRIPTION="A security-aware DNS server"
+HOMEPAGE="http://www.maradns.org/"
+SRC_URI="http://www.maradns.org/download/${PV%.*}/${PV}/${P}.tar.bz2"
```
Comment (juippis) [inline net-dns/maradns/maradns-2.0.17.ebuild]: `https://` whenever possible.

File: net-dns/maradns/maradns-2.0.17.ebuild [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+		"${FILESDIR}"/maradns.service.in > "${T}"/maradns.service \
+		|| die "failed to create the maradns.service file (sed)"
+
+	systemd_dounit "${T}"/maradns.service
+}
+
+pkg_preinst() {
+	enewgroup maradns 99
+	enewuser duende 66 -1 -1 maradns
+	enewuser maradns 99 -1 -1 maradns
```
Comment (juippis) [inline net-dns/maradns/maradns-2.0.17.ebuild]: Please use the new GLEP-81 "acct" way of handling this. You'll have to request UID+GID from the gentoo-dev mailing list, you've probably seen multiple examples how to do it already.

Comment (ktrace) [reply net-dns/maradns/maradns-2.0.17.ebuild]: after 3.4.01(3.4.02), so after #13380

**Comment by juippis** [comment]
0279498e10fabd402b4d3944e4847ace595de0b5

**Pull request closed without merging**